core/primitives/inherent_data.hpp

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_INHERENT_DATA_HPP
+#define KAGOME_INHERENT_DATA_HPP
+
+#include <array>
+#include <map>
+#include <optional>
+#include <vector>
+
+#include <outcome/outcome.hpp>
+#include "common/buffer.hpp"
+
+namespace kagome::primitives {
+
+  using InherentIdentifier = std::array<uint8_t, 8>;
```

Comment:
Better place it into class, otherwise everybody in that namespace who includes that file would see that `using`

Reply:
I meant it to be visible, but perhaps it is actually better to place it inside to make search of it easier.